pkg/kubelet/deviceplugin/manager.go

```diff
@@ -94,9 +96,14 @@ func (m *ManagerImpl) removeContents(dir string) error {
 	return nil
 }
 
+const (
+	// defaultCheckpoint is the file name of device plugin checkpoint
+	defaultCheckpoint = "kubelet_internal_checkpoint"
```

Comment:
This name is too generic. It could be for any module in kubelet. Can we name it `kubelet_device_plugin_ckpt`?